.eslintrc.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
```

Comment:
Holy shit, looks like its been deprecated for at least a year too. How did I miss that? Good to know!